### DISCUSSION TITLE
AzamTV Tanzania Channels working playlist/link

### DISCUSSION THREAD

**Discussion by Gpark02** [iptv-org/community#1153] [Q&A]
Hello Team,
Is there any updated iptv .m3u playlist for Tanzania AzamTV Channels that is working? Please share, and thanks in advance.

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
